src/main/java/me/shedaniel/fiber2cloth/impl/Fiber2ClothImpl.java

```diff
@@ -276,19 +276,18 @@ private void initDefaultFunctionMap() {
             @SuppressWarnings("unchecked") ConfigLeaf<String> leaf = ((ConfigLeaf<String>) node);
             EnumSerializableType type = (EnumSerializableType) leaf.getConfigType();
             ClothSetting.EnumHandler.EnumDisplayOption displayOption = leaf.getAttributeValue(ClothAttributes.SUGGESTION_ENUM, ENUM_DISPLAY_TYPE).orElse(ClothSetting.EnumHandler.EnumDisplayOption.BUTTON);
-            String key = getFieldNameKey(leaf.getName()).getString();
+            Text key = new LiteralText("config." + modId + "." + leaf.getName());
```

Comment:
What's the point in using Text?

Reply:
I switched to Text since TranslatableText was causing issues (which led to this PR), however, perhaps I can go back to String since I'm using a LiteralText now